### ISSUE TITLE
Document how to run parallel tests with the Python extension

### ISSUE THREAD

**Issue by luabud** [microsoft/vscode-docs#2551]
@DonJayamanne  to fill in with details

**Comment by luabud** [comment]
@JoshuaPartlow  Can we update [here](https://code.visualstudio.com/docs/python/unit-testing) to include a subsection "Run tests in parallel" right below the "With PyTest, failed tests also appear in the Problems panel, where you can double-click on an issue to navigate directly to the test" paragraph?

This is how we support running tests in parallel with pytest:


1. Open the integrated terminal and Install ‘pytest-xdist’ (more information can be found on their [documentation](https://pypi.org/project/pytest-xdist/)):
``` 
$ pip install pytest-xdist
```
2. Open the workspace `settings.json` file and add "-n","\<number of CPUs\>" to the pytest arguments list, specifying the number of CPUs you'd like to use. For example: 
```
 "python.testing.pyTestArgs": ["-n", "4"] 
```
After that tests will be run in parallel

**Comment by JoshuaPartlow** [comment]
Documented now with [0d9ff50](https://github.com/microsoft/vscode-docs/commit/0d9ff50e0b9d06b5a39a7b0a537bcacb61709629 ).

**Comment by luabud** [comment]
There's a bug where this is not working properly in some cases. Current workaround is to change ["-n", "4"] to ["-n 4"]. Can we update it in the docs like this for now?

**Comment by DonJayamanne** [comment]
@luabud
Might be best to open a new issue
